vyper/venom/passes/sccp/sccp.py

```diff
@@ -67,14 +67,16 @@ def __init__(self, analyses_cache: IRAnalysesCache, function: IRFunction):
     def run_pass(self):
         self.fn = self.function
         self.dom = self.analyses_cache.request_analysis(DominatorTreeAnalysis)
```

Comment:
i was looking at this and i think technically we don't even need dominator analysis, we just use it for dfs walk